rpc/src/module/test.rs

```diff
@@ -150,44 +143,6 @@ impl IntegrationTestRpc for IntegrationTestRpcImpl {
         self.process_and_announce_block(block_template.into())
     }
 
-    fn broadcast_transaction(&self, transaction: Transaction, cycles: Cycle) -> Result<H256> {
-        let tx: packed::Transaction = transaction.into();
-        let hash = tx.calc_tx_hash();
-        let relay_tx = packed::RelayTransaction::new_builder()
-            .cycles(cycles.value().pack())
-            .transaction(tx)
-            .build();
-        let relay_txs = packed::RelayTransactions::new_builder()
-            .transactions(vec![relay_tx].pack())
-            .build();
-        let message = packed::RelayMessage::new_builder().set(relay_txs).build();
-
-        if let Err(err) = self
-            .network_controller
-            .broadcast(SupportProtocols::Relay.protocol_id(), message.as_bytes())
-        {
-            error!("Broadcast transaction failed: {:?}", err);
-            Err(RPCError::custom_with_error(
-                RPCError::P2PFailedToBroadcast,
-                err,
-            ))
-        } else {
-            Ok(hash.unpack())
-        }
-    }
-
-    fn notify_transaction(&self, tx: Transaction) -> Result<H256> {
```

Comment:
This RPC does not directly push the transaction out. Its purpose is to use the `resumeble_process_tx` interface for testing. I don't think it needs to be deleted?

Reply:
I think I deleted this method by mistake, it shouldn't be impacted by this PR, I've reverted this change, please review it again, thanks.